### DISCUSSION TITLE
Opportunity Activity Rollup

### DISCUSSION THREAD

**Discussion by adam-dalton-ck** [deejay-hub/timeline-lwc#282] [Q&A] [answered]
I wanted to know if it was possible to surface the Activities from the Opportunity records on the Account page or is the component is only capable of displaying the child record data?

**Comment by deejay-hub** [comment, marked as answer]
Hi @adam-dalton-ck have you checked the FAQ guide? For Tasks and Events on the Account object you might find that switching to OpenActivities and ActivityHistory work for you.

Steps here -> https://quip.com/6yvoAcBukqZB#dDRACAgmIoY

Then search for the section called **'What about indirect relationships? The standard Account timeline shows tasks and events not directly linked to the Account. Can Time Warp do something similar?'**

**Reply by adam-dalton-ck** [reply]
Apologies @deejay-hub I didn't realize that FAQ section was related to what I was looking for. Thanks!

**Reply by deejay-hub** [reply]
It might not be :-). Try it and let us know if you get the result you were after.
